jina/__init__.py

```diff
@@ -60,7 +60,7 @@
 
 # do not change this line manually
 # this is managed by proto/build-proto.sh and updated on every execution
-__proto_version__ = '0.0.79'
+__proto_version__ = '0.0.80'
```

Comment:
just curious when will we bump it to 0.1.0?

Reply:
you are right, probably we should have updated to 1.0.0 already. I don't know how the protobuf versions are intended to use.